gym/spaces/graph.py

```diff
@@ -147,8 +103,12 @@ def _sample_sample_space(self, sample_space) -> Optional[np.ndarray]:
         else:
             return None
 
-    def sample(self) -> GraphObj:
-        """Returns a random sized graph space with num_nodes between 1 and 10."""
+    def sample(self) -> NamedTuple:
+        """Generates a single sample graph with num_nodes between 1 and 10 sampled from the Graph.
+
+        Returns:
+            A NamedTuple representing a graph with attributes .nodes, .edges, and .edge_links.
```

Comment:
I think the attributes should be in quotes not as attributes but this doesn't matter much